### ISSUE TITLE
`Wallaby.js Tests` output when using AVA is full of console log output

### ISSUE THREAD

**Issue by danzel** [wallabyjs/public#752]
### Issue description or question

When using AVA as the test framework, the "Wallaby.js Tests" window ends up with a huge console log section, making this area less useful (the console automatically scrolls to the bottom, so I can't see the name of the test that failed)
In comparison, when using mocha there is no `from console` section unless I've output something to the console myself.

Output at the bottom of this issue.

### Wallaby.js configuration file

```javascript
module.exports = function (w) {

	return {
		files: [
			'src/**/*.ts',
			'test/utils/**/*.ts'
		],

		tests: [
			'test/**/*Spec.ts'
		],

		testFramework: 'ava',
		env: {
			type: 'node'
		}
	};
};
```

### Code editor or IDE name and version

Visual Studio Code v1.4.0

### OS name and version

Windows

### Output

```js
​1 failing tests, 0 passing
​ 
​  something [0 ms]​
​  
​  undefined == true
​   c:\Users\dave\Desktop\Code\sinocastel-obd-translator\test\packetDecoderSpec.ts:9:1
​  
​  From console:
​   
​   { passed: false,
​     result: 
​      { title: 'something',
​        fn: [Function],
​        assertions: [ null ],
​        planCount: null,
​        duration: 0,
​        assertError: 
​         { [AssertionError: undefined == true]
​           name: 'AssertionError',
​           actual: undefined,
​           expected: true,
​           operator: '==',
​           message: 'undefined == true',
​           generatedMessage: true },
​        sync: true,
​        contextRef: { context: {} },
​        report: [Function: bound ],
​        threwSync: false,
​        metadata: 
​         { type: 'test',
​           serial: true,
​           exclusive: false,
​           skipped: false,
​           todo: false,
​           failing: false,
​           callback: false,
​           always: false },
​        _timeStart: 1471832384534,
​        _id: 100001,
​        _timeout: 
​         { '0': null,
​           _called: false,
​           _idleTimeout: -1,
​           _idlePrev: null,
​           _idleNext: null,
​           _idleStart: 378142,
​           _onTimeout: null,
​           _repeat: null } },
​     reason: 
​      { [AssertionError: undefined == true]
​        name: 'AssertionError',
​        actual: undefined,
​        expected: true,
​        operator: '==',
​        message: 'undefined == true',
​        generatedMessage: true } }
```

**Comment by ArtemGovorov** [comment]
Thanks for the report. We have accidentally pushed a version with some excessive logging. The issue is fixed in core v1.0.282 (will be updated automatically or you [may force the update](https://wallabyjs.com/docs/intro/troubleshooting.html#forcing-wallabyjs-core-update)).

**Comment by danzel** [comment]
Thanks! :)
